app/services/rbac_api.rb

```diff
@@ -1,32 +1,47 @@
 # frozen_string_literal: true
 
+require 'insights-rbac-api-client'
+
 # This class is meant to handle calls to the RBAC light API
 class RbacApi
+  APPS = 'compliance,inventory'
+  HEADER = { header_params: { 'X_RH_IDENTITY': @b64_identity } }.freeze
```

Comment:
This will never work, the `HEADER` gets evaluated upon load time and then the `@b64_identity˙ is `nil`.